content/attacks/posts/2019-11-27-Upbit.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2019-11-27
+custodians: Upbit
+categories: "Custodian"
+title: "Upbit Loses $49 Million in Cyberattack"
+---
+
+## Summary
+
+On November 27, 2019, Upbit, one of South Korea's largest cryptocurrency exchanges, fell victim to a major cyberattack, resulting in [the theft of approximately $49 million](https://www.coindesk.com/markets/2019/11/27/upbit-is-the-seventh-major-crypto-exchange-hack-of-2019/#:~:text=Upbit%20is%20the%20latest%20hacking,for%20at%20least%20two%20weeks.) worth of Ethereum (342,000 ETH). The attack specifically targeted Upbit's hot wallet, from which the funds were transferred to an unknown wallet address. To safeguard the remaining assets, Upbit promptly [moved them to cold storage](https://www.coindesk.com/markets/2020/01/14/upbit-exchange-resumes-ether-services-months-after-49m-hack/) as a safety measure. The exchange took immediate action, [replenishing the stolen funds with its own assets](https://upbit.com/service_center/notice?id=1085) and ensuring no impact on its customers.
+
+## Attackers
+
+The identity of the attackers involved in the Upbit hack remains unknown. However, the wallet address associated with the attackers is: [0xa09871AEadF4994Ca12f5c0b6056BBd1d343c029](https://etherscan.io/address/a09871aeadf4994ca12f5c0b6056bbd1d343c029).
+
+## Losses
+
+The Upbit hack resulted in the loss of approximately **$49 million** worth of Ethereum (ETH). 
+
+## Timeline:
+
+- **November 27, 2019, 4:06 (UTC):** Upbit [detects a significant security breach](https://upbit.com/service_center/notice?id=1085) targeting its Ethereum hot wallet, leading to the loss of approximately $49 million worth of Ethereum (342,000 ETH)
+- **November 27, 2019, 4:45 (UTC):** Upbit [suspends all cryptocurrency deposits and withdrawals](https://sg.upbit.com/service_center/notice?id=2310), taking immediate action to prevent further unauthorized access and mitigate potential damages.
+- **November 27, 2019, 8:56 (UTC):** Upbit acknowledges the security breach in an official statement, assuring users that their assets will be fully protected and covered by the company.
+- **January 13, 2020, 17:00 (UTC):** Upbit [announces the completion of security enhancements and reopens cryptocurrency deposits and withdrawals](https://sg.upbit.com/service_center/notice?id=2333), restoring full functionality to the exchange.
+
+## Security Failure Causes
+
+The cause of the Upbit hack remains undisclosed, but it is widely believed that the attackers took advantage of weaknesses in Upbit's hot wallet system, leading to unauthorized access and fund transfers. It is speculated that [a combination of phishing and malware attacks]((https://medium.com/@Web3verseAcad/breaking-down-the-upbit-heist-everything-you-need-to-know-556617c31c22)) allowed the hackers to acquire the wallet's private keys.
```

Comment:
Add bullet point

Reply:
Formatted this part in accordance with the latest articles in [dn.insititute](https://dn.institute/attacks/posts/).

```suggestion
**The Wallet System's Vulnerability:** The cause of the Upbit hack remains undisclosed, but it is widely believed that the attackers took advantage of weaknesses in Upbit's hot wallet system, leading to unauthorized access and fund transfers. 
```

```suggestion
**A Phishing-Malware Hybrid Attack:** It is speculated that [a combination of phishing and malware attacks](https://medium.com/@Web3verseAcad/breaking-down-the-upbit-heist-everything-you-need-to-know-556617c31c22) allowed the hackers to acquire the wallet's private keys.
```